### ISSUE TITLE
Can not be installed via  Ansible-Galaxy anmore

### ISSUE THREAD

**Issue by tabacha** [mrlesmithjr/ansible-mariadb-galera-cluster#100]
**Describe the bug**

Our Ansible CI was not able to download the Role via Ansible Galaxy. 

![grafik](https://user-images.githubusercontent.com/1101544/141259212-9bccf69f-c3e3-4dc6-8f31-82801bf30e8a.png)

Since Yesterday it was working.

**To Reproduce**

1. Remove the directory ~/.ansible/roles/mrlesmithjr.mariadb-galera-cluster if it is there.
2. Create a requirements.yml
```yaml
---
roles:
  - name: mrlesmithjr.mariadb-galera-cluster
    src: mrlesmithjr.mariadb-galera-cluster
    version: v0.3.0
```
ansible-galaxy install -r requirements.yml

Gives the following Error:
```
Starting galaxy role install process
- downloading role 'mariadb-galera-cluster', owned by mrlesmithjr
[WARNING]: - mrlesmithjr.mariadb-galera-cluster was NOT installed successfully: - sorry, mrlesmithjr.mariadb-galera-cluster was not found on https://galaxy.ansible.com/api/.
ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
```

**Expected behavior**

It will installed.

**Comment by tabacha** [comment]
Workaround:
```yaml
  - name: mrlesmithjr.mariadb-galera-cluster
    scm: git
    src: https://github.com/mrlesmithjr/ansible-mariadb-galera-cluster
    version: v0.3.3
```

**Comment by mrlesmithjr** [comment]
Apologies for this. I am sure this will affect many folks.

**Comment by tabacha** [comment]
Hi,
when I rename the role in the requirements.yml and change the version number to v0.3.2
```
roles:
  - name: mrlesmithjr.mariadb_galera_cluster
    version: v0.3.2
```

It works. But Versions v0.3.3 is not on ansible galaxy, but on github?
https://galaxy.ansible.com/mrlesmithjr/mariadb_galera_cluster

**Comment by mrlesmithjr** [comment]
This should now be fixed with #101 and thanks for letting me know.
